pipelines/tekton/gitops-update-pipeline/gitops-update-pipeline.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       type: string
     - name: useMergedBuildTestPipeline
       type: string
-      default: "false"
+      default: "true"
```

Comment:
This parameter may be removed now as separate build and test pipelines were removed by https://github.com/opendatahub-io/ai-edge/pull/198. However, this can be done in a separate PR.